frontend/src/__tests__/cypress/cypress/support/e2e.ts

```diff
@@ -18,6 +18,7 @@ import { mockDashboardConfig, mockStatus } from '~/__mocks__';
 import { ODHDashboardConfigModel } from '~/__tests__/cypress/cypress/utils/models';
 import './commands';
 import { addCommands as webSocketsAddCommands } from './websockets';
+import '@cypress/code-coverage/support';
```

Comment:
I want to say that having code coverage enabled does negatively impact run times even if the code is built without code coverage. I ran the tests with and without the cypress code coverage imports and it happened to take longer with the code coverage configured even though I was not intending to collect code coverage.

Unless you can provide some additional data to the alternative, I suggest you conditionally add the cypress code coverage based on the same env variable being set.

Reply:
I found this to disable code coverage hooks: https://github.com/cypress-io/code-coverage?tab=readme-ov-file#disable-plugin

Reply:
Ohh I see , I notice this line `Cannot find coverage file /home/runner/work/odh-dashboard/odh-dashboard/frontend/src/__tests__/cypress/.nyc_output/out.json
[1] Skipping coverage report` when code is built wothout code coverage
Thanks